contracts/tokens/CrowdsaleBodhiToken.sol

```diff
@@ -84,7 +84,7 @@ contract CrowdsaleBodhiToken is BodhiToken {
         uint256 _nativeDecimals, 
         uint256 _decimals) 
         public
-        view 
+        pure 
```

Comment:
what is the difference between `view` and `pure`?

Reply:
`view` is used when the function reads from the contract's state variables. `pure` promises not to read from any of the state variables.